pipeline/splicing/calcVarPriors.py

```diff
@@ -369,11 +381,29 @@ def varInSpliceDonor(variant):
             return True
     return False
 
+def getVarSpliceDonorBounds(variant):
+    '''
+    Given a variant, checks if variant is in a splice donor region
+    If variant is in a splice donor, returns a dictionary with donor region boundaries where variant is located
+    '''
+    inSpliceDonor = varInSpliceDonor(variant)
+    if inSpliceDonor == True:
+        varGenPos = int(variant["Pos"])
```

Comment:
this whole block from line 391-398 is also in `varInSpliceDonor`, `varInSpliceAcceptor`, and `getVarSpliceAcceptorBounds`  -- can you put it into a function to keep DRY? You'll need to take a couple arguments to make it flexible enough to handle donors and acceptors :)

Reply:
I combined varInSpliceDonor and varInSpliceAcceptor into one function and getVarSpliceDonorBounds and getVarSpliceAcceptorBounds into another function. Is that enough to keep it DRY?